### PR TITLE
NEX-707 decode ELT (DT) beacon hex IDs

### PR DESCRIPTION
**Pull request by sam-amsa**
ELT (DT) beacon hex IDs are in a different format to other beacon types, as detailed in Cospas Sarsat specification C/S T.001.

### CODE REVIEW

**Review by skSonar — APPROVED** [review]

**Comment by davidmoten** [comment]
Failing ci, please fix

**Comment by sam-amsa** [comment]
failing ci fixed.

File: src/main/java/au/gov/amsa/fgb/internal/ELTDTLocation.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+public class ELTDTLocation
+    extends BeaconProtocol {
+
+    public static final String ELTDT_PROTOCOL_CODE = "1001";
+    public static final String DEFAULT_COARSE_LATITUDE = "011111111";
+    public static final String DEFAULT_COARSE_LONGITUDE = "0111111111";
+
+    public ELTDTLocation() {
+        super(Lists.newArrayList("00", "10", "?0"), "ELT - ELT-DT Location Protocol", "110");
```
Comment (olamy) [inline src/main/java/au/gov/amsa/fgb/internal/ELTDTLocation.java:19]: what about simply `Arrays.asList` no need to add extra library dependencies

Comment (sam-amsa) [reply src/main/java/au/gov/amsa/fgb/internal/ELTDTLocation.java:19]: Code was copied from another class and the dependency was already there. I didn't even notice this, well spotted. Will fix.

**Pull request merged**